monailabel/config.py

```diff
@@ -49,6 +49,7 @@ class Settings(BaseSettings):
     MONAI_LABEL_DICOMWEB_FETCH_BY_FRAME: bool = False
     MONAI_LABEL_DICOMWEB_SEARCH_FILTER: Dict[str, Any] = {"Modality": "CT"}
     MONAI_LABEL_DICOMWEB_CACHE_EXPIRY: int = 180
+    MONAI_LABEL_DICOMWEB_READ_TIMEOUT = 5.0
```

Comment:
`MONAI_LABEL_DICOMWEB_READ_TIMEOUT: int = 5`

Reply:
Maybe float better? See: https://github.com/encode/httpx/blob/965b8adec3633e1bd249d17fb7a7eed414c74748/httpx/_config.py#L208
```suggestion
    MONAI_LABEL_DICOMWEB_READ_TIMEOUT: float = 5.0
```